app/presenters/tree_node/menu/section.rb

```diff
@@ -0,0 +1,73 @@
+module TreeNode
+  module Menu
+    class Section < Node
+      set_attribute(:key) do
+        "#{@options[:node_id_prefix]}___tab_#{@object.id}"
```

Comment:
I changed to full ids, but forms from users, groups, and roles all seem to go through the same method. Probably should be fixed in a subsequent PR.